src/test/java/com/solvd/BasicHomePageTests.java

```diff
@@ -0,0 +1,32 @@
+package com.solvd;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import com.solvd.pages.HomePage;
+import com.solvd.pages.SearchPage;
+
+//Implementation of Web Automation Java Task 1
+public class BasicHomePageTests extends AbstractTest {
+    @Test
+    public void checkIfHoveringDisplaysMainMenuTest() {
```

Comment:
Consider renaming test methods to follow a consistent and readable format, example testDisplayMainMenuOnHover